dev/sbt-plugin/src/sbt-test/sbt-plugin/distribution/build.sbt

```diff
@@ -53,7 +53,8 @@ InputKey[Unit]("absence") := {
 }
 
 lazy val checkDevRuntimeClasspath = inputKey[Unit]("Checks presence of a jar in the DevRuntime classpath")
-checkDevRuntimeClasspath <<= checkDevClasspathTask(Internal.Configs.DevRuntime)
+
+checkDevRuntimeClasspath := checkDevClasspathTask(Internal.Configs.DevRuntime)
```

Comment:
@TimMoore I have tried `value` and `inputTaskValue` and they result in a slightly different error. IIRC, I can never run the tests locally because of failing external requests.